pkg/controller/common.go

```diff
@@ -28,22 +30,28 @@ type ResourceManager interface {
 type Controller struct {
 	client.Client
 
-	Name    string
-	Version string
+	BaseController controllers.Controller
+	Version        string
 
 	ConfigStore *configstore.Store
 	rm          ResourceManager
 	Log         logr.Logger
 	Scheme      *runtime.Scheme
 }
 
-func NewController(ctx context.Context, name string, rm ResourceManager, config *configstore.Store) *Controller {
+const (
+	networkPoliciesAnnotationName     = "goharbor.io/network-policies"
```

Comment:
Since this annotations can be used by external applications, it should be a Public constant.
I suggest to move it to `apis/meta/v1alpha1`.